lib/wallet/txdb.js

```
@@ -4160,6 +4904,78 @@ class BidReveal extends bio.Struct {
+
+    return this;
+  }
+
+  /**
+   * Instantiate a tx count from a buffer.
+   * @param {Buffer} data
+   * @returns {TXCount}
+   */
+
+  static decode(data) {
+    return new this().decode(data);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)